discover/deployment/articles-dxp/08-liferay-enterprise-search/10-xpack-elasticsearch-6.1/00-xpack-security-61-intro.markdown

```diff
@@ -29,8 +29,7 @@ Here's an overview of using X-Pack to secure the data indexed in Elasticsearch:
     [encryption](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/configuring-tls.html#configuring-tls).
 
 3.  Download and install the 
-[Liferay Enterprise Search Security](https://web.liferay.com/group/customer/dxp/downloads/enterprise-search) 
-[Elastic Stack 6.x]. 
+[Liferay Enterprise Search Security](https://web.liferay.com/group/customer/dxp/downloads/enterprise-search). 
 
 4.  Configure the X-Pack connector with the proper credentials and encryption
```

Comment:
What about the "X-Pack connector"?

Reply:
:disappointed:  Argh, my `find .. -exec sed` wasn't smart enough. I'll go back and do a more thorough search.